docs/install/airgapped-clusters.md

```diff
@@ -1,259 +1,299 @@
 ---
-title: "Airgapped installation"
-weight: 50
+title: "Air-Gap Install"
+linkTitle: "Air-Gap Install"
+weight: 1
 ---
 
-> The following page is for advanced users. The full procedure is estimated to
-> take ~60 minutes to complete.
+## Overview
 
-Clusters without access to the internet require you to explicitly specify the
-resources to be installed. For that reason the `storageos` kubectl plugin
-`--dry-run` flag generates all the yamls for your cluster. You can amend them
-with the images pulled to your private registry. To install Ondat on an
-airgapped cluster, you need
+This guide will demonstrate how to install Ondat onto clusters that don't have direct access to the internet - i.e., [air-gapped](https://en.wikipedia.org/wiki/Air_gap_%28networking%29) environments. Air-gapped environments require cluster administrators to explicitly ensure that Ondat components are locally available before the installation.
 
-- Install the `storageos` kubectl plugin
-- Generate yaml manifests and amend for your use case
-- Pull OCI images to private registries.
-- Install Ondat on your cluster
+> 💡 This guide is recommended for **advanced users** who have experience and permissions to be able to manage air-gapped deployments in their environment. The full procedure for this deployment method is estimated to take ~60 minutes to complete.
 
-There are the following sets of images to pull that will be shown along the
-following steps.
+Below is a quick summary of the procedure that will be covered in this guide;
 
-- the Ondat operator image
-- the images in the StorageOSCluster definition that define the images for each
-  component of the cluster
-- (if applicable) the images to run Etcd as Pods using the Etcd operator
-  deployed by Ondat
+1. Install the Ondat kubectl plugin.
+1. Generate the Ondat deployment manifests for your use case.
+1. Push Ondat container images to your private registry.
+1. Modify the Ondat deployment manifests.
+1. Install Ondat onto your air-gapped cluster.
 
-## Step 1. Install storageos kubectl plugin
+## Prerequisites
 
-```bash
-curl -sSLo kubectl-storageos.tar.gz \
-    https://github.com/storageos/kubectl-storageos/releases/download/v1.1.0/kubectl-storageos_1.1.0_linux_amd64.tar.gz \
-    && tar -xf kubectl-storageos.tar.gz \
-    && chmod +x kubectl-storageos \
-    && sudo mv kubectl-storageos /usr/local/bin/ \
-    && rm kubectl-storageos.tar.gz
-```
+> ⚠️ Make sure you have met the minimum resource requirements for Ondat to successfully run. Review the main [Ondat prerequisites](/docs/prerequisites/) page for more information.
```

Comment:
Make sure you have met the minimum resource requirements for Ondat so your set up would be successful. Review the Prerequisites page for more information.

Reply:
fixed.